### PR TITLE
priyanshu_post added

### PR DESCRIPTION
**Pull request by prix04**
### All Submissions:

* [ ] Have you followed the guidelines in our Contributing document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Comment by KaushikiKrity** [comment]
@AvikantSrivastava The  deployment preview is visible. Please check one.

File: Gemfile.lock [hunk 1]
```diff
@@ -63,6 +63,7 @@ GEM
     sass-listen (4.0.0)
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
+    wdm (0.1.1)
```
Comment (AvikantSrivastava) [inline Gemfile.lock]: Please do not make changes to `Gemfile.lock`
Don't do a `git add` for this file

File: _pages/author-priyanshur.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: "Priyanshu Raj"
+layout: default
+permalink: "/author-priyanshu-raj"
```
Comment (AvikantSrivastava) [inline _pages/author-priyanshur.html:4]: The value of permalink should match with the file name

## Changes to make
1. Either make the permalink from `permalink: "/author-priyanshu-raj"` to `permalink: "/author-priyanshur`
2. Or change the filename from ` _pages/author-priyanshur.html` to ` _pages/author-priyanshu-raj.html`

**Review by AvikantSrivastava — CHANGES_REQUESTED** [review]
Everything else looks fine to me, please make the changes that I have requested :D

File: _pages/author-priyanshur.html (hunk 2 above)
Comment (prix04) [reply _pages/author-priyanshur.html:4]: @AvikantSrivastava changes done, please once review

**Comment by KaushikiKrity** [comment]
@prix04 The changes  look to me ! But your permalink is no working!

File: _pages/author-priyanshur.html (hunk 2 above)
Comment (prix04) [reply _pages/author-priyanshur.html:4]: @kaushikikrityy okay checking!

Comment (prix04) [reply _pages/author-priyanshur.html:4]: @AvikantSrivastava @kaushikikrityy changes done, please check once

**Comment by KaushikiKrity** [comment]
@prix04 The changes looks perfect to me ! @AvikantSrivastava Please review !

**Comment by AvikantSrivastava** [comment]
@kaushikikrityy and @prix04 I see the changes requested are not made yet

File: _pages/author-priyanshur.html (hunk 2 above)
Comment (prix04) [reply _pages/author-priyanshur.html:4]: @AvikantSrivastava  yeah sorry! , I think i have done it partially, gemfile.lock has been deleted as told by you , the other one is not done. Could you please assure me once.

**Comment by KaushikiKrity** [comment]
@AvikantSrivastava as you suggested to keep the value of file and permalink same . We tried that but his permalink was not wring in the deploy preview and  once it changed to previous it is working,

**Comment by AvikantSrivastava** [comment]
image
I see the changes are not made

> @AvikantSrivastava as you suggested to keep the value of file and permalink same . We tried that but his permalink was not wring in the deploy preview and once it changed to previous it is working,

**Comment by AvikantSrivastava** [comment]
image
@prix04 I asked you to make any one change, either change the permalink or change the filename, not both

**Comment by prix04** [comment]
@kaushikikrityy just closing this pr, making a new one.

**Pull request closed without merging**